.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 8
+script:
+  - node_modules/bs-platform/bin/bsb -make-world
```

Comment:
A question, doesn't `npm install` have to be run first or is there some TravisCI opaqueness that does it automatically?  Also what about running `npm run build` instead so it it builds not only the OCaml source but also the SCSS to make sure I don't screw up the SCSS too?  ^.^